gluten-ut/spark33/src/test/scala/io/glutenproject/utils/clickhouse/ClickHouseTestSettings.scala

```diff
@@ -893,7 +894,7 @@ class ClickHouseTestSettings extends BackendTestSettings {
     .exclude("RegexExtract")
     .exclude("RegexExtractAll")
     .exclude("SPLIT")
-    .exclude(" SPARK -34814: LikeSimplification should handleNULL")
+    .exclude("SPARK - 34814: LikeSimplification should handleNULL")
```

Comment:
This test, which did not exclude correctly before, can also pass the test, so maybe we should evaluate whether this test can be opened.

Reply:
Let's keep it excluded.